scripts/templates/detector_vars.tf.j2

```diff
@@ -47,7 +47,7 @@ variable "{{ id }}_disabled_{{ severity }}" {
   default     = null
 }
 
-  {% endfor -%}
+{% endfor -%}
```

Comment:
why did you drop this indent?

Reply:
to avoid adding whitespace at the end and keep the file "terraform fmt"

Reply:
what about using `{%- endfor -%}` for this instead (the - symbol should ignore indent)

Reply:
because if it will ignore the indent for sure it will also remove previous blank line which is desired.

feel free to try, I think it is possible but not without to change the "logic" of the entire "for" block.
Keep in mind it should produce a "terraform fmt-ed" output and consistent in every scenarios (with aggregation / transformation and without them).

while this indent is only here for readability purpose I confess I did not look so far and used the easy way and prefer to avoid too much changes.

the real problem is the lack of testing. adding more test cases in "generator ci" will allow us to trust our changes.

Reply:
ok I added every threshold detectors to tests for generator.

here is the result before this PR changes: https://github.com/claranet/terraform-signalfx-detectors/runs/1349218290?check_suite_focus=true

you can see the problems caused by this endfor indent like:

```
-  variable "requests_errors_5xx_percentage_threshold_major" {
+variable "requests_errors_5xx_percentage_threshold_major" {
```

or 

```
-  resource "signalfx_detector" "real_time_requests_latency" {
+resource "signalfx_detector" "real_time_requests_latency" {
```

now there is the result with this PR changes applied: https://github.com/claranet/terraform-signalfx-detectors/runs/1349338707?check_suite_focus=true

now we could be more confident about submit bigger change. 
That said, I still think this indent is not essential and its deletion is good enough to keep the current logic (which seems good enough too).